invenio_app_ils/circulation/views.py

```diff
@@ -73,6 +77,17 @@ def create_circulation_blueprint(app):
         "/circulation/loans/create", view_func=loan_create, methods=["POST"]
     )
 
+    loan_mail = LoanMailResource.as_view(
+        LoanMailResource.view_name.format(CIRCULATION_LOAN_PID_TYPE),
+        serializers=serializers,
+        ctx=dict(links_factory=loan_links_factory),
+    )
+
+    blueprint.add_url_rule(
+        "{0}/email".format(options["item_route"]),
```

Comment:
I would rename to something more specific, something like `email-overdue`?
Same for the `need_permissions`